File: BookMyShow.Web/bookMyShow/src/app/shared/models/movie.ts
export class Movie {
    id: number;
    name: string;
    language: string;
    duration: string;
    genre: string;
    dateOfRelease: Date;
    constructor(id: number, name: string, language: string, duration: string, genre: string, dateOfRelease: Date
    ) {
        this.id = id;
        this.name = name;
        this.language = language;
        this.duration = duration;
        this.genre = genre;
        this.dateOfRelease = dateOfRelease;
    }
}

File: BookMyShow.API/BookMyShow.DomainModels/Movie.cs
namespace BookMyShow.DomainModels
{
    public class Movie
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Language { get; set; }
        public string Duration { get; set; }
        public string Genre { get; set; }
        public DateTime DateOfRelease { get; set; }
    }
}
